common/transport/amqp/devdoc/amqp_cbs_requirements.md

```diff
@@ -93,8 +93,8 @@ and a body containing `<sasToken>`. **]**
 
 ### events
 
-**SRS_NODE_AMQP_CBS_16_016: [** If either the sender or receiver link emits an `error` event, remaining links shall be detached. **]**
-
+**SRS_NODE_AMQP_CBS_06_001: [** If in the attached state, either the sender or the receiver links gets an error, an error of `azure-iot-amqp-base:error-indicated` will have been indicated on the container object and the cbs will remain in the attached state.  The owner of the cbs MUST detach.
```

Comment:
that does not sound like a test or a test scenario.

Reply:
There is an actual test for this with the text:

remains in the attached state when error emitted to container from link

---
In reply to: [209124407](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209124407) [](ancestors = 209124407)